matecheck.py

```diff
@@ -124,7 +173,15 @@ def analyze_fens(self, fens):
         type=int,
         help="number of threads per position (values > 1 may lead to non-deterministic results)",
     )
-    parser.add_argument("--syzygyPath", help="path to syzygy EGTBs")
+    parser.add_argument(
+        "--syzygyPath", help="path(s) to syzygy EGTBs, with ':' as separator"
+    )
+    parser.add_argument(
+        "--minTBscore",
+        type=int,
+        help="lowest cp score for a TB win",
+        default=20000 - 246,  # for SF this is TB_CP - MAX_PLY
```

Comment:
@vondele Is my assumption that this is the lowest possible TBwin score from SF correct?

Reply:
I think it is ...

Reply:
Not exactly. Internally it is VALUE_TB - MAX_PLY. I do not know how this tranlates to centipawns exactly.